src/helpers/formatters.h

```diff
@@ -42,3 +42,28 @@ struct fmt::formatter<movegen::Move> : fmt::formatter<std::string_view> {
         return fmt::format_to(ctx.out(), "{}{}{}{}", fromC, fromR, toC, toR);
     }
 };
+
+/*
+ * helper type to specialize a fmt print
+ * use as eg:
+ * fmt::print("score {}", ScorePrint(score)); -> eg: "score mate 8"
+ */
+struct ScorePrint {
+    int32_t score;
```

Comment:
Patch was a little old, I'll fix before merging :)
```suggestion
    Score score;
```